packages/rest/src/providers/invoke-method.provider.ts

```diff
@@ -19,3 +20,29 @@ export class InvokeMethodProvider implements Provider<InvokeMethod> {
     return route.invokeHandler(this.context, args);
   }
 }
+
+@bind(
+  asMiddleware({
+    group: 'invokeMethod',
```

Comment:
So, looks like `group` is to LB4, what middleware phase was to LB3, right? @raymondfeng

Reply:
yes.